### PR TITLE
dev, v3.0: change tidb-cluster deploy doc

### PR DESCRIPTION
**Pull request by xiaojingchen**
### What is changed, added or deleted? 
changed docs:
- dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md
- v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md

main fixes:
1. add configuration doc link
2. change the struct of doc

### CODE REVIEW

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 1]
```diff
@@ -16,74 +16,10 @@ namespace="tidb"
...
-Helm 安装完成后，通过下面命令获取要安装的 tidb-cluster chart 的 `values.yaml` 配置文件：
-
-{{< copyable "shell-regular" >}}
-
-```shell
-mkdir -p /home/tidb/${releaseName} && \
-helm inspect values pingcap/tidb-cluster --version=${chartVersion} > /home/tidb/${releaseName}/values-${releaseName}.yaml
-```
-
 > **注意：**
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: 这里的“注意”是承接着上文的内容，请一并挪到下面相应的位置上。

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 2]
```diff
@@ -16,74 +16,10 @@ namespace="tidb"
...
-Helm 安装完成后，通过下面命令获取要安装的 tidb-cluster chart 的 `values.yaml` 配置文件：
-
-{{< copyable "shell-regular" >}}
-
-```shell
-mkdir -p /home/tidb/${releaseName} && \
-helm inspect values pingcap/tidb-cluster --version=${chartVersion} > /home/tidb/${releaseName}/values-${releaseName}.yaml
-```
-
 > **注意：**
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: 同上。

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 3]
```diff
@@ -100,9 +36,22 @@ TiDB Operator 把 `values.yaml` 作为 TiDB 集群的配置文件。`values.yaml
...
+Helm 安装完成后，通过下面命令获取要安装的 tidb-cluster chart 的 `values.yaml` 配置文件：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+mkdir -p /home/tidb/${releaseName} && \
+helm inspect values pingcap/tidb-cluster --version=${chartVersion} > /home/tidb/${releaseName}/values-${releaseName}.yaml
+```
+
+有关配置信息请参考：[TiDB 集群部署配置文档](/reference/configuration/tidb-in-kubernetes/cluster-configuration/)
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
有关配置信息请参考 [TiDB 集群部署配置文档](/reference/configuration/tidb-in-kubernetes/cluster-configuration.md)。
```

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 4]
```diff
@@ -100,9 +36,22 @@ TiDB Operator 把 `values.yaml` 作为 TiDB 集群的配置文件。`values.yaml
...
+
 ## 部署 TiDB 集群
 
-TiDB Operator 和 Helm 部署完成并配置完成，可以通过下面命令部署 TiDB 集群：
+TiDB Operator 部署完成并配置完成，可以通过下面命令部署 TiDB 集群：
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
TiDB Operator 部署并配置完成后，可以通过下面命令部署 TiDB 集群：
```

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 5]
```diff
@@ -100,9 +36,22 @@ TiDB Operator 把 `values.yaml` 作为 TiDB 集群的配置文件。`values.yaml
...
+Helm 安装完成后，通过下面命令获取要安装的 tidb-cluster chart 的 `values.yaml` 配置文件：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+mkdir -p /home/tidb/${releaseName} && \
+helm inspect values pingcap/tidb-cluster --version=${chartVersion} > /home/tidb/${releaseName}/values-${releaseName}.yaml
+```
+
+有关配置信息请参考：[TiDB 集群部署配置文档](/reference/configuration/tidb-in-kubernetes/cluster-configuration/)
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
有关配置信息请参考 [TiDB 集群部署配置文档](/reference/configuration/tidb-in-kubernetes/cluster-configuration.md)。
```

File: v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 6]
```diff
@@ -100,9 +36,22 @@ TiDB Operator 把 `values.yaml` 作为 TiDB 集群的配置文件。`values.yaml
...
+
 ## 部署 TiDB 集群
 
-TiDB Operator 和 Helm 部署完成并配置完成，可以通过下面命令部署 TiDB 集群：
+TiDB Operator 部署完成并配置完成，可以通过下面命令部署 TiDB 集群：
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]:
```suggestion
TiDB Operator 部署并配置完成后，可以通过下面命令部署 TiDB 集群：
```

**Comment by yikeke** [comment]
**Important Update**: We have added a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

Click `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure and fix all the issues in your PR. It is recommended to install "markdownlint" extension in your VS Code editor, so it will check all potential issues that fail the CI check. 

@xiaojingchen If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

Ref: https://github.com/pingcap/docs-cn/pull/1494

**Comment by xiaojingchen** [comment]
what does MD031 mean?

**Comment by lilin90** [comment]
@xiaojingchen It means [this rule](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines).

**Pull request merged**